ddtrace/contrib/django/db.py

```diff
@@ -99,7 +99,8 @@ def _trace(self, func, sql, params):
         )
 
         with span:
-            span.set_tag(sqlx.QUERY, sql)
+            # The sql query MUST NOT be added as a tag as is added by the agent in the tag `sql.query`.
```

Comment:
Is there any side effect here of setting other than "no reason since the agent does it?"